<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>722</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="ScreenWidget" name="screenWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>400</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="CurrentSourceInfoWidget" name="currentSourceInfoWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>36</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>36</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>865</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QDockWidget" name="sceneDockWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="SceneWidget" name="sceneWidget">
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>200</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="sourceDockWidget">
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>226</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="SourceWidget" name="sourceWidget">
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>200</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="audioMixerDockWidget">
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>226</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="AudioMixerWidget" name="audioMixerWidget">
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>200</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="controlDockWidget">
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>226</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="ControlWidget" name="controlWidget">
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>200</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SceneWidget</class>
   <extends>QWidget</extends>
   <header location="global">DockWidget/SceneWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SourceWidget</class>
   <extends>QWidget</extends>
   <header location="global">DockWidget/SourceWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AudioMixerWidget</class>
   <extends>QWidget</extends>
   <header location="global">DockWidget/AudioMixerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ControlWidget</class>
   <extends>QWidget</extends>
   <header location="global">DockWidget/ControlWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScreenWidget</class>
   <extends>QWidget</extends>
   <header location="global">ScreenWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CurrentSourceInfoWidget</class>
   <extends>QWidget</extends>
   <header location="global">CurrentSourceInfoWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
